power_grid_model_c/power_grid_model/include/power_grid_model/container.hpp

```diff
@@ -35,7 +35,7 @@ inline constexpr size_t get_index_bool_array(std::array<bool, N> arr, size_t idx
 
 template <typename U, typename First, typename... Rest>
 constexpr size_t get_type_index() {
-    if constexpr (std::is_same<U, First>::value) {
+    if constexpr (std::is_same_v<U, First>) {
```

Comment:
just a matter of taste but you could also use `std::same_as<U, First>` here. it's not exactly the same but for most use cases it is. saves another `_v`. Don't worry about it too much